src/core/eme/handle_session_events.ts

```diff
@@ -156,52 +174,56 @@ export default function handleSessionEvents(
         messageEvent
       );
 
-      const getLicense$ = Observable.defer(() => {
+      const getLicense$ = observableDefer(() => {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary cast. ([typescript:S4325](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4325))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=236&issues=AWP-ZPtRuoX1TGVkGlQV&open=AWP-ZPtRuoX1TGVkGlQV)

[ISSUE_KEY]: AWP-ZPtRuoX1TGVkGlQV